src/sentry/incidents/models.py

```diff
@@ -524,51 +525,29 @@ class Meta:
 
 
 @region_silo_only_model
-class AlertRuleTriggerAction(Model):
+class AlertRuleTriggerAction(AbstractNotificationAction):
     """
     This model represents an action that occurs when a trigger is fired. This is
     typically some sort of notification.
     """
 
+    # Aliases from NotificationAction
+    Type = ActionService
+    TargetType = ActionTarget
```

Comment:
I noticed that this adds an extra service to the enum. Probably fine, but maybe we should guard against creating `AlertRuleTriggerActions` of type `SENTRY_NOTIFICATION`

Reply:
Added an exemption list and a test to double check